src/OmenAgentResultMapping.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.24;
+
+import {Prediction} from "./structs.sol";
+
+contract OmenAgentResultMapping {
+    event PredictionAdded(
+        address indexed marketAddress,
+        uint16 estimatedProbabilityBps,
+        address indexed publisherAddress,
+        bytes32 txHash,
```

Comment:
To double check, Hugo can access txHashes on Dune to use them to link trades and these predictions?

Reply:
Yes